### ISSUE TITLE
Lack of some API calls

### ISSUE THREAD

**Issue by szymor** [radekwielonski/warsaw-data-api#6]
First of all, thank you for the effort of creating this API. I use it for my indoor time table monitor. I have some proposals for further development:

- it would be nice to have an explicit call for getting busstopId - the possible use case is when we would like to get timetables for different lines of the same bus stop (multiple calls of API of id e923fa0e-d96c-43f9-ae6e-60518c9f3238), in such a case we do not need to convert the name of the bus stop twice (by calling API of id b27f4c17-5c50-4a5b-89dd-236b282bc499),
- it would be nice to implement an API call of id 88cd555f-6f31-43ca-9de4-66c479ad5942 (get lines for the given bus stop id) - right now if we want to check time tables for the bus stop, we need to hardcode bus line numbers.

That's all from my side. :)

**Comment by radekwielonski** [comment]
Hey, thanks for the input, please let me know if you would be interested in contributing and creating PR with that functionalities? 

If it's not really needed ASAP I can implement this when I will have free time.

**Comment by szymor** [comment]
Rather not, I use your API mostly for prototyping things. Current functionality is completely sufficient for my case, I do not expect change of bus line numbers in the nearest future. Treat my issue rather as a suggestion and appreciation of your work.

If I change my mind though, e.g. I decide to develop my own private app even further, I could consider contribution to your API as well.

**Comment by radekwielonski** [comment]
Hey, I've implemented method get_lines_for_bus_stop_id, for the second subsection. Could you please let me know what exactly did you mean by getting busstopId in the first subsection?

**Comment by szymor** [comment]
The idea is simple. Currently you do sth like that for each call:
```python
def get_bus_stop_schedule_by_name(
        self, bus_stop_name: str, bus_stop_nr: str, line: str
    ) -> ZtmSchedule:
        query_params: Dict[str, Union[str, int, None]] = {
            "id": "b27f4c17-5c50-4a5b-89dd-236b282bc499",
            "apikey": self.apikey,
            "name": bus_stop_name,
        }
        response = self.__get_data_from_ztm(self.schedule_endpoint, query_params)
        clean_response = convert_list_to_dict(response[0]["values"])
        return self.get_bus_stop_schedule_by_id(
            clean_response["zespol"], bus_stop_nr, line
        )
```
It would be nice to have an explicit API for b27... in order to use the returned ID in API calls that accept bus stop IDs (example: Szczęśliwice bus stop has ID 4119). Otherwise, we are forced to use API twice for each call by name (once with `__get_data_from_ztm`, once with `get_bus_stop_schedule_by_id`).

**Comment by radekwielonski** [comment]
Ok, so that means creating something like get_bus_stop_details_by_name, right?

**Comment by szymor** [comment]
Or rather `get_bus_stop_id_from_name`. I do not think you can get multiple IDs or more data from a single name using API call of id b27..., but that needs to be verified.

**Comment by radekwielonski** [comment]
Resolved in version 0.5.3
